pages/admin/request/[id].tsx

```diff
@@ -95,6 +96,7 @@ const Request: NextPage<Props> = ({ id: idString }: Props) => {
           {type === 'REPLACEMENT' && (
             <ReasonForReplacementCard applicationId={id} editDisabled={reviewRequestCompleted} />
           )}
+          {type !== 'REPLACEMENT' && <AdditionalInformationCard applicationId={id} />}
```

Comment:
We want to integrate the card component in #172 to avoid conflicts, so we should not integrate it in this PR

Reply:
Addressed!